src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -193,6 +208,7 @@ export function isNeverType(type: Type): type is NeverType {
 export function getInputType(
     context: SdkContext,
     type: Type,
+    format: string | undefined,
```

Comment:
what is the difference between `format?: string` and `format: string | undefined`?
we might need to align with line 109

Reply:
@ArcturusZhang  If you define format?: string, it is optional parameter, it must be put in the last one. Here It is not good to pass as the last one.